src/pytest_html/scripts/dom.js

```diff
@@ -90,6 +90,10 @@ const dom = {
             if (['image', 'video'].includes(format_type)) {
                 media.push({ path: content, name, format_type })
             }
+
+            if (format_type === 'html') {
+              resultBody.querySelector('.extraHTML').insertAdjacentHTML('beforeend', `<div>${content}</div>`)
```

Comment:
The reason I do it this way is that I want the HTML to be in the order it was appended.